Jwt.php

```diff
@@ -122,6 +127,34 @@ public function verifyToken(Token $token)
 
         $signer = Yii::createObject($this->supportedAlgs[$alg]);
 
-        return $token->verify($signer, $this->key);
+        return $token->verify($signer, $this->prepareKey());
+    }
+    
+    /**
+     * Detects key file path and resolves Yii alias if given.
+     * @return string|null
+     * @throws \LogicException when file path does not exist or is not readable
+     */
+    public function prepareKey()
+    {
+        $keyPath = null;
+
+        if (strpos($this->key, '@') === 0) {
+            $keyPath = 'file://' . Yii::getAlias($this->key);
+        }
+
+        if (strpos($this->key, 'file://') === 0) {
+            $keyPath = $this->key;
+        }
+
+        if ($keyPath !== null) {
+            if (!file_exists($keyPath) || !is_readable($keyPath)) {
```

Comment:
This check is already present here https://github.com/lcobucci/jwt/blob/3.2/src/Signer/Key.php#L61